srcpkgs/linux-vt-setcolors/patches/8.diff

```diff
@@ -0,0 +1,15 @@
+diff --git Makefile Makefile
+index 7a9cc3e..ad5ad1d 100644
+--- Makefile
++++ Makefile
+@@ -1,7 +1,9 @@
+ PREFIX=/usr/local
++CC=gcc
```

Comment:
You don't need to specify `CC`, `make` can decide on one automatically.

Reply:
Ah, my upstream patch was apparently a little wrong.  I'll get this fixed up!